peewee_extra_fields/__init__.py

```diff
@@ -1153,6 +1153,32 @@ def python_value(self, value: str) -> namedtuple:
         return value
 
 
+class EnumField(CharField):
+    """
+    This class enables a Enum like field for Peewee.
+    """
+    db_field = "enum"
+
+    def __init__(self, choices, *args, **kwargs):
+        if not issubclass(choices, Enum):
+            raise TypeError("Argument choices must be of type enum.")
+        self.choices = choices
+        super().__init__(*args, **kwargs)
+
+    def db_value(self, value):
+        return value.name
+
+    def python_value(self, value):
+        return self.choices(value)
+
+    def get_column_type(self):
+        return "enum"
+
+    def coerce(self, value):
+        if value not in self.choices:
+            raise Exception("Invalid Enum Value `{}".format(value))
```

Comment:
Use F-String:

```python
raise Exception(f"Invalid Enum Value {value}")
```

Reply:
I love python 3.6! If you say we target that python version, great.
(You will also have noticed. I used py3 style super())

Reply:
All the code uses F-String, because is faster. :grinning: 

![f-strings](https://user-images.githubusercontent.com/1189414/38151554-2e7612d2-343a-11e8-9449-d68d414c0066.png)

Reply:
Yeah. That is the best thing fast to type and execute. Really great addition to python, those f-strings.